serveradmin/resources/views.py

```diff
@@ -113,19 +113,24 @@ def index(request):
     per_page = int(request.GET.get(
         'per_page', request.session.get('resources_per_page', 8)))
     request.session['resources_per_page'] = per_page
-    hosts_pager = Paginator(list(hosts.values()), per_page)
+
+    try:
+        hosts_pager = Paginator(list(hosts.values()), per_page).page(page)
+    except (PageNotAnInteger, EmptyPage):
```

Comment:
`EmptyPage` .. what about queries which actually results into no entries?

Reply:
I am resetting the page to 1 in case the term changes or the DB data has changed an the user tries to access a non existent page anymore.

Let me just say that I am not too happy with the current implementation at all of this view because of a few things:
- We use Query which does not support pagination so always all objects are loaded
- Servershell search auto completion is missing here
- The method is quite big and should be split into more functions

So I will later tackle this view again but want to keep it as it is for now.

Reply:
With my other comment https://github.com/innogames/serveradmin/pull/82#discussion_r402255885 feel free and do it in a way that it is just safe. ;)